sdks/python/apache_beam/examples/complete/game/leader_board.py

```diff
@@ -331,10 +331,13 @@ def run(argv=None):
         'user': 'STRING',
         'total_score': 'INTEGER',
     }
+
+    def format_user_score_sums(user_score):
+      return {'user': user_score[0], 'total_score': user_score[1]}
+
     (events  # pylint: disable=expression-not-assigned
      | 'CalculateUserScores' >> CalculateUserScores(args.allowed_lateness)
-     | 'FormatUserScoreSums' >> beam.Map(
-         lambda (user, score): {'user': user, 'total_score': score})
+     | 'FormatUserScoreSums' >> beam.Map(format_user_score_sums)
```

Comment:
Why is this change needed? (Similarly for other lambda function changes.)

Reply:
Too long line after the unpacking change.

Reply:
Why don't we break into two lines then?

Reply:
multi-line lambdas http://www.artima.com/weblogs/viewpost.jsp?thread=147358